examples/evolution_strategy.py

```diff
@@ -0,0 +1,109 @@
+"""Generates CMA-ES animation on sphere function.
+
+For gradient-based optimizers (inheriting from :class:`GradientOptBase`):
+* ``adam``: :class:`AdamOpt`
+* ``gradient_ascent``: :class:`GradientAscentOpt`
+
+For evolution strategies (inheriting from :class:`EvolutionStrategyBase`):
+* ``cma_es``: :class:`CMAEvolutionStrategy`
+* ``sep_cma_es``: :class:`SeparableCMAEvolutionStrategy`
+* ``lm_ma_es``: :class:`LMMAEvolutionStrategy`
+* ``openai_es``: :class:`OpenAIEvolutionStrategy`
+
+Usage:
+    # Generate frames.
+    python evolution_strategy.py <sphere or rastrigin> <evolution_strategy_name>
+
+    # Stitch everything together.
+    ffmpeg -r -6 -i "es_output/<evolution_strategy_name>_itr_%*.png" es_output/<evolution_strategy_name>
+"""
+from pathlib import Path
+
+import fire
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import tqdm
+
+from ribs.emitters.opt import _get_es
+
+mpl.rcParams["font.weight"] = "light"
```

Comment:
You can take out these `rcParams` -- they're just for making the figure look pretty